web/react/components/signup_team_complete.jsx

```diff
@@ -42,11 +42,15 @@ WelcomePage = React.createClass({
             state.email_error = "";
         }
 
-        client.signupTeam(email, this.props.state.team.name,
+        client.signupTeam(email,
```

Comment:
Nope, if you scroll up a bit in the file you can see that this is for a new email over the one that the user originally chose